jsapp/js/projects/projectViews/projectsFilterEditor.tsx

```diff
@@ -75,11 +79,14 @@ export default function ProjectsFilterEditor(props: ProjectsFilterEditorProps) {
     return fieldDefinition.availableConditions.map(
       (condition: FilterConditionName) => {
         const conditionDefinition = FILTER_CONDITIONS[condition];
-        return {label: conditionDefinition.label, id: conditionDefinition.name};
+        return {label: conditionDefinition.label, value: conditionDefinition.name};
       }
     );
   };
 
+  const isCountryFilterSelected = () =>
```

Comment:
Turn this into simple const value instead of function. This is a functional component - each render runs whole function, so there is no need for a helper funcion so simple as this :)